stock_available_mrp/tests/test_potential_qty.py

```diff
@@ -510,6 +510,6 @@ def test_potential_qty__list(self):
         )
 
         self.assertEqual(
-            {p1.id: 3.0, p2.id: 3.0, p3.id: 0.0},
+            {p1.id: 0.0, p2.id: 3.0, p3.id: 0.0},
```

Comment:
was that a bug in the previous version?

Reply:
@Cedric-Pigeon however travis seems to be red because of this... https://travis-ci.org/OCA/stock-logistics-warehouse/jobs/213350781#L1307